index.html

```diff
@@ -1,23 +1,79 @@
+			</p><p>
+				Apart from lacking the convenience of a web interface, this process also puts considerable demands on the code contributions: the mails are expected to be plain text only (no HTML!), for example, and the diffs embedded in the mails must apply cleanly (no whitespace changes!), among other things.
+			</p><p>
+				A few tutorials out there try to help with this process (e.g. <a href="https://github.com/git-for-windows/git/blob/master/CONTRIBUTING.md">Git for Windows' detailed instructions how to contribute patches to the Git project</a>). GitGitGadget tries a different approach: allow contributing patches to the Git project itself by opening a Pull Request at https://github.com/gitgitgadget/git and let GitGitGadget prepare and send the corresponding mails.
+			</p>
+		</div>
+		
+		<h2>How can you use GitGitGadget?</h2>
+		<div class="block">
+			<p>
+				So you cloned https://github.com/git/git and implemented a bug fix or a new feature? And you already pushed it to your own fork? Good, now is the time to direct your web browser to https://github.com/gitgitgadget/git and to open a Pull Request. Please make sure to use a descriptive Pull Request title and description; GitGitGadget will use these as the cover letter.
+			</p><p>
+				The first time you use GitGitGadget, you need to be added to the list of users with permission to use GitGitGadget (this is a <i>very</i> simple anti-spam measure). Any user who is already on that list can do that, by adding a comment to that Pull Request that says <tt>/allow &lt;username&gt;</name></tt> (with your GitHub login name).
+			</p><p>
+				The Pull Request will trigger a few Checks, most importantly one that will build Git and run the test suite on the main platforms, to make sure that everything works as advertised. 
+			</p><p>
+				Once everything is ready to go, add a comment to that Pull Request saying <tt>/submit</tt>. This will trigger GitGitGadget (you can see the progress via the Check called "GitGitGadget PR Handler"): it will wrap your Pull Request into a nice bundle of mails in the format expected on the Git mailing list.
+			</p>
+		</div>
 
-		<p>
-		GitGitGadget's idea is for users to open a Pull Request at <a href="https://github.com/gitgitgadget/git">https://github.com/gitgitgadget/git</a> with a good description of their patch series. Then, the command <tt>/submit</tt>, issued via a comment on said PR will tell GitGitGadget to send the patches as mail thread to <a href="mailto:mailto:git@vger.kernel.org">the Git mailing list</a>, with the Pull Request's description as cover letter.
+		<h2>What happens after GitGitGadget sent the mails?</h2>
+		<div class="block">
+			<p>
+				The patches will be reviewed by volunteers (be gentle...) and you will most likely receive helpful comments. It is a good idea to wait at least one day (time zones, you know?) for reviewer comments to trickle in before sending another iteration of your patch series (if needed).
```

Comment:
The paranthetical statements here are well-meaning, but a bit casual. Consider rewriting. For instance

> It is a good idea to wait at least one day (time zones, you know?) for reviewer comments...

could be 

> The Git developer community is globally distributed, so please wait a day or two for reviewer comments...

Reply:
Done. Any more suggestions like this? (I'm a non-native speaker, and it shows...)